autopush/endpoint.py

```diff
@@ -253,22 +253,25 @@ def _router_fail_err(self, fail):
         fail.trap(RouterException)
         exc = fail.value
         if isinstance(exc, RouterException):
-            if exc.log_exception or exc.status_code >= 500:
-                fmt = fail.value.message or 'Exception'
-                self.log.failure(format=fmt,  # pragma nocover
-                                 failure=fail, status_code=exc.status_code,
-                                 errno=exc.errno or "",
-                                 client_info=self._client_info)
-            if 200 <= exc.status_code < 300:
-                self.log.info(format="Success", status_code=exc.status_code,
-                              logged_status=exc.logged_status or "",
-                              client_info=self._client_info)
-            elif 400 <= exc.status_code < 500:
-                self.log.info(format="Client error",
-                              status_code=exc.status_code,
-                              logged_status=exc.logged_status or "",
-                              errno=exc.errno or "",
-                              client_info=self._client_info)
+            if exc.log_exception:
```

Comment:
oh, so log_exception is only about the failure call -- but this changes it to be about any kind of logging (thank you test coverage gap for showing me this)